README.md

```diff
@@ -11,53 +11,19 @@ If you are new to SEMP, you can learn more here for full documentation and API r
 
 These tutorials apply to the SEMP API starting in version 2. For older versions of the SEMP API, you can see the [Legacy SEMP Documentation](http://docs.solace.com/SEMP/Using-Legacy-SEMP.htm)
 
-**Note**: Currently SEMP v2 is only supported on Solace message router appliances running SolOS version 7.2.2 and above. Solace virtual message router will add SEMP v2 support in the near future.
+**Note**: SEMP v2 is only supported on Solace virtual message routers and Solace message router appliances running SolOS version 7.2.2 and above.
 
 ## Contents
 
-This repository contains code and matching tutorial walk throughs for different basic scenarios. It is best to view the associated [tutorials home page](https://solacesamples.github.io/solace-samples-semp/).
+This repository contains code and matching tutorial walk throughs for different basic scenarios and programming languages. It is best to view the associated [tutorials home page](https://solacesamples.github.io/solace-samples-semp/).
```

Comment:
Could we point this link to the Dev Portal: http://dev.solace.com/get-started/semp-tutorials/